services/places/functions/list/main.py

```diff
@@ -0,0 +1,85 @@
+import json
+import boto3
+import base64
+from decimal import Decimal
+
+PAGE_SIZE = 5
+AWS_REGION = "us-east-1"
+
+dynamodb = boto3.resource('dynamodb',region_name=AWS_REGION)
+places_table= dynamodb.Table('places')
+
+
+def list_places(event):
+    
+    queryStringParameters = event["queryStringParameters"]
+    
+    items = {
+        'Items' : []
+    }
+    
+    if ((queryStringParameters) and ('last_evaluated_key' in queryStringParameters) and
+        (queryStringParameters['last_evaluated_key'])):
```

Comment:
is better if this key is same that other lambda, for not generate confusion in the API